prs-mapper.lua

```diff
@@ -72,40 +74,14 @@ end
 
 local function make_room()
     local info = map.room_info
-    local coords = {0,0,0}
+    local coords = {info.x,-info.y,0}
     addRoom(info.vnum)
-	  setRoomName(info.vnum, info.name)
+    setRoomName(info.vnum, info.name)
     local areas = getAreaTable()
     local areaID = areas[info.area]
     if not areaID then
         areaID = addAreaName(info.area)
-    else
-        coords = {getRoomCoordinates(map.prev_info.vnum)}
-        local shift = {0,0,0}
-        for k,v in pairs(info.exits) do
-            if v == map.prev_info.vnum and move_vectors[k] then
-                shift = move_vectors[k]
-                break
-            end
-        end
-        for n = 1,3 do
-            coords[n] = coords[n] - shift[n]
-        end
-        -- map stretching
-        local overlap = getRoomsByPosition(areaID,coords[1],coords[2],coords[3])
-        if not table.is_empty(overlap) then
-            local rooms = getAreaRooms(areaID)
-            local rcoords
-            for _,id in ipairs(rooms) do
-                rcoords = {getRoomCoordinates(id)}
-                for n = 1,3 do
-                    if shift[n] ~= 0 and (rcoords[n] - coords[n]) * shift[n] <= 0 then
-                        rcoords[n] = rcoords[n] - shift[n]
-                    end
-                end
-                setRoomCoordinates(id,rcoords[1],rcoords[2],rcoords[3])
-            end
-        end
```

Comment:
remove map stretching conflict -- also unneeded in grid format of PR -- with new lookahead mini-mapping thingy... not sure what to call it